services/data-info/src/data_info/routes.clj

```diff
@@ -51,7 +51,8 @@
     avus-routes/avus-routes
     exists-routes/existence-marker
     filetypes-routes/filetypes-operations
-    users-routes/permissions-gatherer
+    permission-routes/permissions-gatherer
+    permission-routes/data-item-permissions
```

Comment:
This seemed like a clearer option to me because it allows me to give more descriptive names to the route definitions